component-library/component-lib/src/lib/form-components/checkbox/checkbox.component.ts

```diff
@@ -136,14 +135,19 @@ export class CheckboxComponent
    * @param isDisabled
    */
   setDisabledState(isDisabled: boolean) {
-    if(isDisabled){
+    if (isDisabled) {
       this.formGroup.get(this.config.id)?.disable();
-    }else{
+    } else {
       this.formGroup.get(this.config.id)?.enable();
     }
   }
 
   ngOnInit() {
+    const retControl = this.config.formGroup.get(this.config.id);
```

Comment:
Why are you setting a local formControl value rather than rely on the value in the config directly? I see this in a lot of the components, but the code is new.